pkg/apis/kops/v1alpha1/cluster.go

```diff
@@ -251,6 +254,24 @@ type ClusterSpec struct {
 	Assets *Assets `json:"assets,omitempty"`
 }
 
+// FileAssetSpec defines the structure for a file asset
+type FileAssetSpec struct {
+	// Name is a shortened reference to the asset
+	Name string `json:"name,omitempty"`
+	// Path is the location this file should reside
+	Path string `json:"path,omitempty"`
```

Comment:
Would you be OK with putting the assets all into the same directory i.e. dropping this field?  I'm thinking then we don't have to worry about immutable base images, and we also could in theory deliver the assets via another mechanism, i.e. just like configmap in a pod.